extensions/amp-a4a/rtc-documentation.md

```diff
@@ -156,10 +156,11 @@ The RTC Response to a GET request must meet the following requirements:
 
 *   Status Code = 200
 *   Headers (in addition to automatically added headers):
-    *   CORS
-    *   AMP-Access-Control-Allow-Source-Origin
-    *   Access-control-allow-origin
-    *   Access-control-expose-header: AMP-Access-Control-Allow-Source-Origin
+    *   Following [CORS headers](https://developer.mozilla.org/en-US/docs/Glossary/CORS): 
```

Comment:
Can we just link to https://github.com/ampproject/amphtml/blob/master/spec/amp-cors-requests.md?